.github/workflows/build_all.yml

```diff
@@ -176,50 +162,42 @@ jobs:
         env:
           DEBIAN_META_INFO: deb.distribution=focal-ci;deb.component=main;deb.architecture=amd64
         run: |
+          set -euo pipefail
```

Comment:
Are we sure we didn't somehow rely on not failing in the past?

Reply:
Will summarize my understanding in order to find an answer ... The step does three things
* find version of created magma debian artifact for integration test workflow
* upload created artifacts
* create firebase output from upload response
The step can be seen as failed (imho) if no magma artifact was created or if the upload fails - but the question is what failed mean here: I assume we always want to create the firebase output, i.e., the current restriction might be to hard. Suggestion: lets create the firebase output as failed before the `set -euo pipefail` and override it in the follow up steps.

Reply:
changed - small test if the override of the output works as expected: https://github.com/nstng/magma/actions/workflows/aaa_test_output.yml